tests/installation.bats

```diff
@@ -61,4 +61,9 @@ function teardown {
     CURL="curl --fail -H 'Host: ${NAME}.ddev.site' --silent --output /dev/null --url $URL"
     echo "# curl: $CURL" >&3
     ${CURL}
+
+    echo "# Testing switch_branch.sh"
```

Comment:
Should switch_branch be in its own test case?

Reply:
We could do something like `echo "echo 'some debug text';" >> index.php` to test the resetting.

Reply:
Yes, switch_branch can be its own, but each test case requires full, full setup, which is why I cheated. Thanks to this review of bats, I know how to do tests with different requirements next time and it will be better :) I' afraid to be working too much longer with this one.

I did manually do a change to README to test and saw the expected results. I agree it's a good test case, but again, can't consider doing everything on this round.